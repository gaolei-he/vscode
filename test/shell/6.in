94
/cba/dca/bac
/caa/bd
/acd/b/b
/bb/bd/dab
/bd/cd/cdd
/cab/adc
/ad/b/ba
/a/da/dba
/cdc
/bd
/ac/ca/cb
/a/dd/da
/aa
/bdc
/c/ddb/db
/cc/bdb/dc
/aac/d
/a
/cad
/b/a/d
/a/c/ab
/da
/c/a
/d
/cd/cd/dbb
/da/bc/db
/ad
/cc/d
/bbb/ac
/cbc/aab/dd
/a/c
/bbb/a/c
/bda
/db
/bbb/bbd/b
/a
/ca/ba
/cb/cda
/acb
/ddb/bc/d
/b/cd/cbd
/cd
/bca/adc/caa
/cb/bc/adc
/cda/a/bdd
/cbc/ac
/cb
/bc
/aab
/d
/aab
/dac
/cc/dcb/cd
/cca
/acd/ab
/b/abd
/bad
/ba
/cb
/b/ccd
/cdd/a
/d
/d/bb
/ab/bba
/dba
/bc/db/aca
/bba
/dbc/cbc
/d/aab/dcc
/b/d/bd
/ba/d
/dc/bda/c
/bb/a/ac
/cbb
/acc/a
/c
/c/c/bdb
/a
/cd/d
/dd/bbd/daa
/aa
/dc/ca
/ac/ac
/a/caa/dda
/a
/ba/ac/bab
/b/b
/d/abc/ada
/cbb
/d/bcb
/cda
/dd/b
/a/d/bd
/bbb
5
pwd
cd ..
ls
pwd
pwd
