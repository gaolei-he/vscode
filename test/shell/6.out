/
a
aa
aab
aac
ab
ac
acb
acc
acd
ad
b
ba
bad
bb
bba
bbb
bc
bca
bd
bda
bdc
c
ca
caa
cab
cad
cb
cba
cbb
cbc
cc
cca
cd
cda
cdc
cdd
d
da
dac
db
dba
dbc
dc
dd
ddb
/
/
